crates/crevice/src/glsl.rs

```diff
@@ -11,7 +11,11 @@ pub struct Dimension<A, const N: usize> {
 /// Type-level linked list terminator for array dimensions.
 pub struct DimensionNil;
 
-/// Trait for type-level array dimensions. Probably shouldn't be implemented outside this crate.
+/// Trait for type-level array dimensions.
+///
+/// # Safety
+///
+/// Probably shouldn't be implemented outside this crate.
```

Comment:
lol

We could make it a sealed trait.

Reply:
I did not want to change too much from upstream crevice, refactoring the doc comment to match the lint seemed better for that

Reply:
Yup until we come to a final decision on whether we (1) use upstream crevice (2) use vendored crevice with additional upstream prs merged (3) fork crevice indefinitely (4) build our own thing, I don't want to touch this code at all.

Reply:
Should we then instead just ignore all of the lints for this crate instead?

Reply:
I'm open to both solutions (and agree that ignoring lints is probably the better forward facing fix), but I won't block on that.

Reply:
The most important thing is to not meaningfully change the code itself. As long as we aren't doing that, I don't care too much.

Reply:
I ignored clippy in crevice